README.md

```diff
@@ -1,326 +1,12 @@
 # Migrating from Embedding JSAPI v1 or v2 to Embedding API v3 (Migration Guide)
 
-> The initial release of the Embedding API library (version 3.0) was made available as part of the Tableau 2021.4 launch. To find out about using the current version of the library (now 3.4), see the documentation for the [Tableau Embedding API v3 documentation](https://help.tableau.com/current/api/embedding_api/en-us/index.html).
+## Documentation
 
+For further information about the Tableau Embedding API v3 release, please refer to the Tableau help content:
 
+* [Tableau Embedding API v3 documentation](https://help.tableau.com/current/api/embedding_api/en-us/index.html)
+* [Upgrade from Tableau JavaScript API v1 and v2](https://help.tableau.com/current/api/embedding_api/en-us/docs/embedding_api_migration_guide.html)
```

Comment:
I don't feel strongly so feel free to ignore, but the only thought that came to mind was we might want to make this link to the new migration guide more prominent. SEO for "tableau embedding api migration guide" places this readme higher than the new one (at least on Google; Bing places the new one higher).

For example, swap the order of the bullets or something else like:

> The Embedding API V3 migration guide has moved to [Upgrade from Tableau JavaScript API v1 and v2](https://help.tableau.com/current/api/embedding_api/en-us/docs/embedding_api_migration_guide.html).
> 
> For further information about the Tableau Embedding API v3 release, please refer to the [Tableau Embedding API v3 documentation](https://help.tableau.com/current/api/embedding_api/en-us/index.html).

Reply:
Great ideas, thanks!